### DISCUSSION TITLE
Nuclide & Growth Models - MOOSE

### DISCUSSION THREAD

**Discussion by KavyaV28** [idaholab/moose#17727] [Q&A Modules: General] [answered]
Hello, I'm a new user and I'm currently working on building a nuclide and growth model to simulate the release of recrystallization energy from irradiated PTFE. I was wondering if anyone knew about any papers that discuss building nuclides growth models in the MOOSE framework.

**Comment by permcody** [comment]
@laagesen - This might be a good question for you to advise on.

**Comment by laagesen** [comment, marked as answer]
Hi @KavyaV28 - 
In case you had not seen it, we do a have some documentation pages on the nucleation capabilities in the MOOSE framework:

https://mooseframework.inl.gov/modules/phase_field/Nucleation/DiscreteNucleation.html
https://mooseframework.inl.gov/modules/phase_field/Nucleation/LangevinNoise.html

We have done more with the discrete nucleation approach in the past. @amjokisaari implemented a discrete nucleation approach during her Ph.D. 
https://www.sciencedirect.com/science/article/pii/S0927025615006564?via%3Dihub
This is not exactly the same as, but most similar to, the DiscreteNucleationForce approach currently available in MOOSE. Additionally this article may be of interest:
https://www.sciencedirect.com/science/article/pii/S0927025621000963?via%3Dihub

**Reply by jessecarterMOOSE** [reply]
Thanks @laagesen . I hadn't seen that last reference yet (it's from the future!).

Mind if I ask a question about these two papers? Both have an example where nucleation happens throughout the simulation (not just at time 0). In @amjokisaari 's paper, the fitted slope in the Avrami equation is about 2, but in the newer paper, it is about 3. The newer paper says it should be 3 for this case and that it is 2 when nucleation occurs at t=0 and then demonstrates that in an example too. Any idea why there is a discrepancy? Does it have to do with how the timing of the nucleation events is chosen?

**Reply by laagesen** [reply]
@amjokisaari can probably answer that better than me

**Reply by KavyaV28** [reply]
@laagesen Thank you
